swap_meet/decor.py

```diff
@@ -1,2 +1,21 @@
-class Decor:
-    pass
\ No newline at end of file
+import uuid
+from swap_meet.item import Item
+
+class Decor(Item):
+    def __init__(self, id = None, width = 0, length = 0, condition = 0.0):
+        super().__init__(id, condition)
+        self.width = width
+        self.length = length
+
+    def get_category(self):
```

Comment:
This check is not necessary since only Decor instances can call this method. You can just return "Decor".